examples/analytics-3p-remote-frame-helper.js

```diff
@@ -0,0 +1,193 @@
+/**
+ * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * This provides the "glue" between the AMP Analytics tag and the third-party
+ * vendor's metrics-collection page.
+ *
+ * The vendor's page must implement onNewAmpAnalyticsInstance().
+ */
+class AmpAnalyticsRemoteFrameManager {
+  constructor() {
+    /**
+     * @type {function<!string>}
```

Comment:
...I think you may be reviewing an old version of the code. This region of this file currently looks like:
class AmpAnalytics3pRemoteFrameHelper {
  constructor(win) {
    /**
     * @type {!Array<function(string)>}
     * @private
     */
    this.eventsListeners_ = [];